src/components/ResultContainer.tsx

```diff
@@ -12,7 +12,17 @@ function ResultContainer({ result }: { result: Result[] | null }) {
     ? result.length === download.length && result.every((r) => download.includes(r.dataset_uuid))
     : false;
 
-  // TODO: deal with erros
+  function summaryStats() {
+    let datasets = 0;
+    let subjects = 0;
+    if (result) {
+      result.forEach((item) => {
+        datasets += 1;
+        subjects += item.num_matching_subjects;
+      });
+    }
+    return `Summary stats: ${datasets} datasets, ${subjects} subjects`;
+  }
 
```

Comment:
This code will run every time the component renders anyway, so let's lift this out of the function and put it directly into the plain JS part to make that clear!

Reply:
i.e. have a `const summaryText = ...` variable that contains all this text